Marlin/src/module/settings.cpp

```diff
@@ -3763,6 +3763,9 @@ void MarlinSettings::reset() {
       }
     #else
       planner.set_advance_k(ADVANCE_K);
+      #if ENABLED(ADVANCE_K_EXTRA)
+        EXTRUDER_LOOP() other_extruder_advance_K[e] = ADVANCE_K;
```

Comment:
Good catch. The setter should be called so the fixed point vars are also updated

Reply:
oh not here.